### PR TITLE
[IMP] barcodes_generator_product: clarify documentation

### PR DESCRIPTION
**Pull request by HekkiMelody**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
clarified instructions to set up basic EAN13 pattern (will port to upper versions)

File: barcodes_generator_product/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1,12 +1,10 @@
 This module expands Odoo functionality, allowing user to generate barcode
 depending on a given barcode rule for Products.
 
-For example, a typical pattern for products is  "20.....{NNNDD}" that means
-that:
-* the EAN13 code will begin by '20'
+For example, a typical pattern for products is "8012345....." that means that:
```
Comment (legalsylvain) [inline barcodes_generator_product/readme/DESCRIPTION.rst]: Dont remove variable pattern that is valid setting. But ypu could add your exemple. 

Thanks !

**Review by legalsylvain — CHANGES_REQUESTED** [review]

File: barcodes_generator_product/readme/DESCRIPTION.rst (hunk 1 above)
Comment (HekkiMelody) [reply barcodes_generator_product/readme/DESCRIPTION.rst]: Thanks for your feedback, I left the previous example and also added the new one.

**Review by francesco-ooops — APPROVED** [review]
LGTM

@legalsylvain good for you? :)

**Review by legalsylvain — COMMENTED** [review]
/ocabot mère patch

Thanks !

(You can port that change in head version if you want)

**Comment by francesco-ooops** [comment]
> /ocabot mère patch

the mother of all patches 😎

sure we'll forward port it

**Comment by rvalyi** [comment]
> /ocabot mère patch
> 
> Thanks !
> 
> (You can port that change in head version if you want)

\ocabot ta mère toi-même !

cc @legalsylvain :-D

**Comment by legalsylvain** [comment]
I hate auto-correction on my cell phone... ;-)

**Pull request merged**